### PR TITLE
[mds-web-sockets] Add WS Client Push -> NATS Publish

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Adds capability for WS clients to push entities through the websocket connection, which will then fan out via NATS. Eventually we'll need to add things like per-entity RBAC, etc.. but for the time being, we just allow all socket-wide authenticated clients to push any supported entities.

Note: No tests were added, because the existing tests already exercised the eternally accessible API. Just now we're using NATS as a middle-man, so for the send/receive tests, we have WS Client -> WS Server -> NATS -> WS Server -> WS Client, where as previously we had WS Client -> WS Server -> WS Client. Unfortunately, we don't have a nice way for testing NATS locally, so we're mocking it here. But we can see the mocks are used as a result of the message handlers in the test suite :)

```
> AUTH%Bearer REDACTED_TOKEN 
< AUTH%{"status":"Success"}
> SUB%event
< SUB%{"status":"Success"}
> PUSH%event%{"foo": "bar"}
< event%{"foo":"bar"}
```

## 📦 Impacts:
- mds-web-sockets

## ✅ PR Checklist
- [x] Test in deployed cluster

### CODE REVIEW

**Comment by avatarneil** [comment]
D'oh, the tests don't spin up a local NATS... Let me move to draft for now.

File: packages/mds-web-sockets/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "start": "yarn watch server",
     "test": "yarn test:eslint && yarn test:unit",
     "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "DOTENV_CONFIG_PATH=../../.env nyc --lines 50 ts-mocha --project ../../tsconfig.json --exit",
+    "test:unit": "DOTENV_CONFIG_PATH=../../.env jest --forceExit",
```
Comment (drtyh2o) [inline packages/mds-web-sockets/package.json:42]: `--forceExit` 😱

**Review by drtyh2o — COMMENTED** [review]
There is a new version of `ws` available ([7.4.0](https://github.com/websockets/ws/releases/tag/7.4.0)). I didn't upgrade to it in my PR because of this WIP, but perhaps you want to pick it up along with these changes.

File: packages/mds-web-sockets/package.json (hunk 1 above)
Comment (avatarneil) [reply packages/mds-web-sockets/package.json:42]: Yeah... I need to add a ProcessManager wrapper to the WS service, right now there's no good way to safe-close it. Wanna save that for another PR though (but it's certainly on the bucket list of things I want to enhance for WS in Q4)

Comment (avatarneil) [reply packages/mds-web-sockets/package.json:42]: Resolved in https://github.com/lacuna-tech/mds-core/pull/487

**Review by janedotx — APPROVED** [review]
LGTM

**Review by nessur — APPROVED** [review]

**Pull request merged**